src/Illuminate/Foundation/Console/Kernel.php

```diff
@@ -294,6 +299,33 @@ protected function bootstrappers()
     }
 
     /**
+     * Set environment if 'env' option if provided.
+     *
+     * @param  InputInterface  $input
+     * @return void
+     */
+    protected function setEnvironment($input)
+    {
+        if (! app()->runningInConsole()) {
```

Comment:
This function can never be used in the core. Please just call `$this->app->...`.